tap_learndash/streams.py

```diff
@@ -1,46 +1,755 @@
+        th.Property("topic_per_page_custom", th.IntegerType),
+        th.Property("price_type", th.StringType),
+        th.Property("price_type_paynow_price", th.StringType),
+        th.Property("price_type_subscribe_price", th.StringType),
+        th.Property("price_type_closed_price", th.StringType),
+        th.Property("price_type_closed_custom_button_url", th.StringType),
+        th.Property("prerequisite_enabled", th.BooleanType),
+        th.Property("prerequisite_compare", th.StringType),
+        th.Property("prerequisites", th.ArrayType(th.StringType)),
+        th.Property("points_enabled", th.BooleanType),
+        th.Property("points_access", th.StringType),
+        th.Property("points_amount", th.StringType),
+        th.Property("progression_disabled", th.BooleanType),
+        th.Property("expire_access", th.BooleanType),
+        th.Property("expire_access_days", th.IntegerType),
+        th.Property("expire_access_delete_progress", th.BooleanType)
+    ).to_dict()
+
+    def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
+        """Return a context dictionary for child streams."""
+        return {
+            "course_id": record["id"]
+        }
+
+
+class CoursesPrerequisitesStream(LearnDashStream):
+    """Define custom stream."""
+    name = "courses_prerequisites"
+    path = "/sfwd-courses/{course_id}/prerequisites"
+    primary_keys = ["course_id", "id"]
+    parent_stream_type = CoursesStream
+    ignore_parent_replication_keys = True
+    schema = th.PropertiesList(
+        th.Property("course_id", th.IntegerType),
+        th.Property("id", th.IntegerType),
+        th.Property("date", th.DateTimeType),
+        th.Property("date_gmt", th.DateTimeType),
+        th.Property("guid", th.ObjectType(
+            th.Property("rendered", th.StringType)
+        )),
+        th.Property("modified", th.DateTimeType),
+        th.Property("modified_gmt", th.DateTimeType),
+        th.Property("slug", th.StringType),
+        th.Property("status", th.StringType),
+        th.Property("type", th.StringType),
+        th.Property("link", th.StringType),
+        th.Property("title", th.ObjectType(
+            th.Property("rendered", th.StringType)
+        )),
+        th.Property("content", th.ObjectType(
+            th.Property("protected", th.BooleanType),
+            th.Property("rendered", th.StringType)
+        )),
+        th.Property("author", th.IntegerType),
+        th.Property("featured_media", th.IntegerType),
+        th.Property("menu_order", th.IntegerType),
+        th.Property("template", th.StringType),
+        th.Property("categories", th.ArrayType(th.StringType)),
+        th.Property("tags", th.ArrayType(th.StringType)),
+        th.Property("ld_course_category", th.ArrayType(th.IntegerType)),
+        th.Property("ld_course_tag", th.ArrayType(th.StringType))
+    ).to_dict()
+
+
+class CoursesUsersStream(LearnDashStream):
+    """Define custom stream."""
```

Comment:
Remove the `define custom stream` docstring for these taps, or replace with documentation from the API.